pom.xml

```diff
@@ -56,7 +56,7 @@
         <!-- Dependencies versions -->
         <apache.commons.version>1.9.0</apache.commons.version>
         <commons.io.version>2.11.0</commons.io.version>
-        <farao.dependencies.version>1.27.0</farao.dependencies.version>
+        <farao.dependencies.version>1.28.0-SNAPSHOT</farao.dependencies.version>
```

Comment:
farao dependencies 1.28.0 has now been released, either use it or switch to 1.29.0-SNAPSHOT for testing purposes